content/blog/2018/egress-performance/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Egress Gateway Performance Investigation
+description: Verifies the performance impact of adding an egress-gateway.
+publishdate: 2018-12-12
+subtitle: An Istio Egress Gateway performance assessment
+attribution: Jose Nativio, IBM
+weight: 77
+keywords: [performance,traffic-management,egress,mongo]
+---
+
+The main objective of this investigation was to determine the impact on performance and resources utilization when an egress gateway is added in the service mesh to access an external service (MongoDB, in this case). The steps to configure an egress gateway for an external MongoDB are described in the blog [Consuming External MongoDB Services](/blog/2018/egress-mongo/).
+
+The application used for this investigation was the Java version of Acmeair, which simulates an airline reservation system. This application is used in the Performance Regression Patrol of Istio daily builds, but on that setup the microservices have been accessing the external MongoDB directly via their sidecars (istio-proxies), without an egress gateway.
+
+The diagram below illustrates how regression patrol currently runs with Acmeair and Istio:
+
+{{< image width="70%" ratio="60%"
+    link="./acmeair_regpatrol3.png"
+    caption="Acmeair benchmark in the Istio performance regression patrol environment"
+    >}}
+
+Another difference is that the application communicates with the external DB with plain MongoDB protocol. The first change made for this study was to establish a TLS communication between the MongoDB and its clients running within the application, as this is a more realistic scenario.
+
+Several cases for accessing the external database from the mesh were tested and described next.
+
+## Egress Traffic Cases
```

Comment:
Here and elsewhere, only capitalize the first letter of headers (unless using a proper noun or acronym)